src/networkremote/client.h

```diff
@@ -0,0 +1,35 @@
+#ifndef CLIENT_H
+#define CLIENT_H
+
+#include <QObject>
+#include <QTcpSocket>
+#include "incomingmsg.h"
+#include "outgoingmsg.h"
+#include "core/player.h"
+
+class Application;
+
+class NetworkRemoteClient : public QObject
+{
+  Q_OBJECT
+public:
+  explicit NetworkRemoteClient(Application *app, QObject *parent = nullptr);
+  ~NetworkRemoteClient();
+  void Init(QTcpSocket*);
+  QTcpSocket* GetSocket();
```

Comment:
Formatting wrong here to:
```suggestion
  QTcpSocket *GetSocket();
```

Reply:
Changed